src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAResolvePackageAssetsTask.cs

```diff
@@ -161,7 +161,7 @@ public void It_warns_on_invalid_culture_codes_of_resources(string tfm, bool shou
             var invalidContextWarnings = engine.Warnings.Where(msg => msg.Code == "NETSDK1188");
             invalidContextWarnings.Should().HaveCount(shouldHaveWarnings ? 1 : 0);
 
-            var invalidContextMessages = engine.Messages.Where(msg => msg.Code == "NETSDK1188");
+            var invalidContextMessages = engine.Messages.Where(msg => msg.Code == "NETSDK1187" && msg.Importance == MessageImportance.Low);
```

Comment:
The error code here changed.  Was the test broken before?

Reply:
This was a bad copy/paste on my part - I'll fix that now.